modules/storage/storage-account/main.bicep

```diff
@@ -1,446 +1,723 @@
+metadata name = 'Azure Storage Account'
+metadata description = 'This Bicep module creates a Storage Account with zone-redundancy, encryption, virtual network access, and TLS version.'
+metadata owner = 'dciborow'
+
 @description('Deployment Location')
-param location string
+param location string = resourceGroup().location
```

Comment:
I'd suggest updating the description to mention that the location defaults to the resource group location.

Reply:
updated